.github/workflows/tests.yaml

```diff
@@ -47,21 +47,36 @@ jobs:
       # the next job.
       #
       # via https://github.community/t/how-to-use-env-with-container-image/17252/25
-      - name: Save the Docker image name to a variable so we can share and re-use it in other jobs via `${{ needs.deploy-test-synapse-image.outputs.docker_image_name }}`
+      - name: Save the Docker image name to a variable so we can share and re-use it in other jobs via `${{ needs.build-test-synapse-image.outputs.docker_image_name }}`
```

Comment:
Removing the tag from the end of the name.

We now specify this explicitly set this in the `services` which still works 🤷